.github/workflows/stalled.yml

```diff
@@ -0,0 +1,20 @@
+name: Close Stalled PRs
+on:
+  schedule:
+    - cron: '15 15 * * *' # Run every day at 15:15 UTC / 7:15 PST / 8:15 PDT
+permissions:
+  pull-requests: write
```

Comment:
I think this will need the token to perform write operations as follows - 

```
 - name: GitHub App token
        id: github_app_token
        uses: tibdex/github-app-token@v1.5.0
        with:
          app_id: ${{ secrets.APP_ID }}
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
          installation_id: 22958780

      - name: Stale PR
        uses: actions/stale@v8
        with:
          repo-token: ${{ steps.github_app_token.outputs.token }}
          ...
          ...
```